docs/deployment/openshift.md

```diff
@@ -155,8 +155,11 @@ $ oc process -f cronjob.yml --param JOB_NAME=index-run \
 
 ### Upgrade your instance
 
-If you have performed some changes to your instance (e.g. configuration) or you want to upgrade the version of the charts, you can do so with
-the `upgrade` command of `helm`. Note that you still need to disable the openapi validation which is yet not supported in version 3.1.2 (However, it is merged into the master branch and should come out soon):
+If you have performed some changes to your instance (e.g. configuration) or you want to upgrade the version of the chart, you can do so with
+the `upgrade` command of `helm`.
+
+!!! warning "Not supported yet"
+    Note that you still need to disable the openapi validation which is yet not supported in version 3.1.2 (However, it is merged into the master branch and should come out soon). For now we will have to `helm uninstall` and then install again.
```

Comment:
```suggestion
    Note that you still need to disable the openapi validation which is as of yet not supported in version 3.1.2 (However, it is merged into the master branch and should come out soon). For now we will have to `helm uninstall` and then install again.
```

Reply:
I do not understand this code suggestion :( "as of" seems redundant there, no?

Reply:
If we want the "yet" before "not supported", that's the way to go, but you're right that we can just use "yet" if we move it:

```suggestion
    Note that you still need to disable the openapi validation which is not supported yet in version 3.1.2 (However, it is merged into the master branch and should come out soon). For now we will have to `helm uninstall` and then install again.
```